node/cn/handler.go

```diff
@@ -998,7 +998,7 @@ func handleReceiptsRequestMsg(pm *ProtocolManager, p Peer, msg p2p.Msg) error {
 		// Retrieve the requested block's receipts, skipping if unknown to us
 		results := pm.blockchain.GetReceiptsByBlockHash(hash)
 		if results == nil {
-			if header := pm.blockchain.GetHeaderByHash(hash); header == nil || header.ReceiptHash != types.EmptyRootHash {
+			if header := pm.blockchain.GetHeaderByHash(hash); header == nil || !header.EmptyReceipts() {
```

Comment:
it's little bit weird that EmptyReceipts() returns true if the receipthash == EmptyHash.
What about 'IsEmptyReceipts' or 'IsReceiptEmpty'?

Reply:
This `func EmptyXX() bool` functions are used in several data structures in Klaytn codebase (e.g. `StateDB.Empty()`, `txList.Empty()`). If it's not too confusing, I would leave it as is.

Reply:
@blukat29 Okay. got it.